packages/server/__snapshots__/form_submissions_spec.coffee

```diff
@@ -12,33 +24,56 @@ Started video recording: /foo/bar/.projects/e2e/cypress/videos/abc123.mp4
   2 passing
 
 
-  (Tests Finished)
+  (Results)
 
-  - Tests:           2
-  - Passes:          2
-  - Failures:        0
-  - Pending:         0
-  - Skipped:         0
-  - Duration:        10 seconds
-  - Screenshots:     0
-  - Video Recorded:  true
-  - Cypress Version: 1.2.3
+  ┌───────────────────────────────────────────────────┐
+  │ Tests:        2                                   │
+  │ Passing:      2                                   │
+  │ Failing:      0                                   │
+  │ Pending:      0                                   │
+  │ Skipped:      0                                   │
+  │ Screenshots:  0                                   │
+  │ Video:        true                                │
+  │ Duration:     X seconds                           │
+  │ Spec Ran:     form_submission_passing_spec.coffee │
+  └───────────────────────────────────────────────────┘
 
 
   (Video)
 
   - Started processing:   Compressing to 32 CRF
-  - Finished processing:  /foo/bar/.projects/e2e/cypress/videos/abc123.mp4 (0 seconds)
+  - Finished processing:  /foo/bar/.projects/e2e/cypress/videos/abc123.mp4 (X seconds)
+
+
+====================================================================================================
+
+  (Run Finished)
 
 
-  (All Done)
+      Spec                                                Tests  Passing  Failing  Pending  Skipped 
+  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
+  │ ✔ form_submission_passing_spec.coffee          Xs        2        2        -        -        - │
+  └────────────────────────────────────────────────────────────────────────────────────────────────┘
+    All specs passed!                              Xs        2        2        -        -        -  
```

Comment:
Shouldn't this just be, 'All tests passed!'

Reply:
In the failure state it describes the number of specs that failed `(1 of 5) 20%` so that's why the success state is written in the perspective of specs, not tests.